estate/views/estate_menus.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<odoo>
+    <menuitem id="estate_menu_root" name="Real Estate">
+        <menuitem id="estate_menu_advertisements" name="Advertisements">
+            <menuitem id="estate_action_properties" action="action_estate_properties"/>
+        </menuitem>
+        <menuitem id="estate_menu_settings" name="Settings">
+            <menuitem id="estate_action_property_types" action="action_estate_property_types"/>
+            <menuitem id="estate_action_property_tags" action="action_estate_property_tags"/>
+        </menuitem>
+    </menuitem>
```

Comment:
for the exercise: make one line per menuitems, not using encapsulation
but using parent_id instead.
this will help you when adding menuitems on bridge modules or modules depending on the module where you want to put a new menu item